src/core/server/config/deprecation/core_deprecations.ts

```diff
@@ -599,7 +599,6 @@ const logFilterDeprecation: ConfigDeprecation = (
     addDeprecation({
       configPath: 'logging.filter',
       level: 'critical',
-      documentationUrl: `https://github.com/elastic/kibana/blob/${branch}/src/core/server/logging/README.mdx#loggingfilter`,
```

Comment:
The replacement is configuring logging for specific contexts and we don't have a 1-to-1 replacement for the setting. I thought it best to remove the doc link  rather than direct folks to the dev logging README that is a little outdated by now.

Reply:
I think RewriteAppender is the closest substitute we have, but we also explicitly say it is experimental, so +1 on removing the link for now.